src/tests/Equinor.ProCoSys.IPO.Query.Tests/GetLatestMdpIpoStatusOnCommPkgs/GetLatestMdpIpoStatusOnCommPkgsQueryHandlerTests.cs

```diff
@@ -0,0 +1,187 @@
+                    "MDP Title",
+                    "Description2",
+                    DisciplineType.MDP,
+                    new DateTime(),
+                    new DateTime(),
+                    null)
+                {
+                    MeetingId = meetingId
+                };
+
+                _mdpInvitation2 = new Invitation(
+                    TestPlant,
+                    _projectName,
+                    "MDP Title 2",
+                    "Description3",
+                    DisciplineType.MDP,
+                    new DateTime(),
+                    new DateTime(),
+                    null)
+                {
+                    MeetingId = meetingId
+                };
+
+                _mdpInvitation1.AddCommPkg(commPkg1);
+                _mdpInvitation2.AddCommPkg(commPkg2);
+
+                context.Invitations.Add(_mdpInvitation);
+                context.SaveChangesAsync().Wait();
+
+                var timeProvider = new ManualTimeProvider(new DateTime(2020, 2, 2, 0, 0, 0, DateTimeKind.Utc));
+                TimeService.SetProvider(timeProvider);
+
+                context.Invitations.Add(_mdpInvitation1);
+                context.Invitations.Add(_mdpInvitation2);
+                context.SaveChangesAsync().Wait();
+                _mdpInvitationId1 = _mdpInvitation1.Id;
+                _mdpInvitationId2 = _mdpInvitation2.Id;
+            }
+        }
+
+        [TestMethod]
+        public async Task HandleGetInvitationsByCommPkgNoQuery_ShouldReturnOkResult()
+        {
+            using (var context = new IPOContext(_dbContextOptions, _plantProvider, _eventDispatcher, _currentUserProvider))
+            {
+                var query = new GetLatestMdpIpoStatusOnCommPkgsQuery(new List<string> {_commPkgNo1, _commPkgNo2}, _projectName);
+                var dut = new GetLatestMdpIpoStatusOnCommPkgsQueryHandler(context);
+                var result = await dut.Handle(query, default);
+
+                Assert.AreEqual(ResultType.Ok, result.ResultType);
+            }
+        }
+
+        [TestMethod]
+        public async Task HandleGetInvitationsByCommPkgNoQuery_ShouldReturn2Invitations()
+        {
+            using (var context = new IPOContext(_dbContextOptions, _plantProvider, _eventDispatcher, _currentUserProvider))
+            {
+                var query = new GetLatestMdpIpoStatusOnCommPkgsQuery(new List<string> { _commPkgNo1, _commPkgNo2 }, _projectName);
+                var dut = new GetLatestMdpIpoStatusOnCommPkgsQueryHandler(context);
+
+                var result = await dut.Handle(query, default);
+
+                Assert.IsNotNull(result);
+                Assert.AreEqual(ResultType.Ok, result.ResultType);
+
+                var commPkgsWithMdpIposDtos = result.Data;
+                Assert.AreEqual(2, commPkgsWithMdpIposDtos.Count);
+
+                Assert.IsTrue(commPkgsWithMdpIposDtos.SingleOrDefault(i => i.LatestMdpInvitationId == _mdpInvitationId1) != null);
```

Comment:
Assert.IsNotNull(commPkgsWithMdpIposDtos.SingleOrDefault(i => i.LatestMdpInvitationId == _mdpInvitationId1));

same in line below and line 181 (elsewhere?)